src/main/java/com/uber/cadence/internal/worker/WorkerShutDownHandler.java

```diff
@@ -0,0 +1,70 @@
+/*
+ *  Modifications Copyright (c) 2017-2020 Uber Technologies Inc.
+ *  Portions of the Software are attributed to Copyright (c) 2020 Temporal Technologies Inc.
```

Comment:
Since this is a brand new file that we are adding without being inspired by Temporal, you can remove this copyright note.

Please also change 2020 to 2022.

Reply:
done